settings.gradle

```diff
@@ -75,6 +75,7 @@ include {
         integrations {
             _ 'integrations-retrofit-adapter'
             _ 'integrations-jackson-module'
+            _ 'integrations-kotlinx-coroutines'
```

Comment:
I think this should belongs under `Arrow Fx KotlinX`

```
        fx {
            _ 'fx'
            _ 'fx-mtl'
            _ 'fx-rx2'
            _ 'fx-reactor'
            _ 'fx-kotlinx'
        }
```

Reply:
oh, fair enough 👍